integrations/gcp/.port/spec.yaml

```diff
@@ -10,6 +10,11 @@ configurations:
     type: string
     require: false
     description: Base64 encoding of your Google Cloud's ADC Credentials configuration file. For more information, see the <a href="https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/cloud-providers/gcp/installation" target="_blank">Google documentation</a>
+  - name: searchAllResourcesPerMinuteQuota
+    type: integer
+    require: false
+    description: The allowed number of requests to the GCP assets inventory. By default it is set to 400, but can be adjusted per to your GCP plan. For more information see the <a href="https://cloud.google.com/asset-inventory/docs/quota#project_quota" target="_blank">GCP quota documentation</a>
+    default: 400
 deploymentMethodRequirements:
```

Comment:
is this still needed?

Reply:
Yes, This is for non-terraform installations

Reply:
the `searchAllResourcesPerMinuteQuota` var, is what is used should the integration fail to fetch cloud quota api for any of the services, would you rather I have it defined as a constant in the code instead ?

Reply:
A more granular approach in my opinion would be to allow optionally setting quota on each kind (service) to ingest via the `port-app-config`. 
That way, without having to activate cloud quota api, one can easily infer the allowed quota's for the service api and add it to the port app config specifications, the integration then utilizes these parameters to control rate limits per kind

Reply:
This limit isn't really applying to the hard limit made by GCP. I agree this will be much cleaner, but considering we are hard limited by GCP we cannot allow two different limitations to exist, one logical and another actual limit

Reply:
lets have it as an environment variable for now, with a default of 400. 
And do you use the same client for real time events as well?